docs/docs/tutorial.md

```diff
@@ -21,7 +21,7 @@ It'll also have a few neat features:
 
 ### Want to skip all this and just see the source?
 
-[It's all on GitHub.](https://github.com/reactjs/react-tutorial)
+[It's all on GitHub](https://github.com/reactjs/react-tutorial). Or, [view and edit your own live copy on HyperDev](https://hyperdev.com/#!/remix/react_tutorial/0539a08d-bcda-458e-8647-94813e4248b4).
```

Comment:
Yes, that's it - the link creates a clone of the latest version of React tutorial project (https://hyperdev.com/#!/project/shade-king), so anyone who clicks on it will get their own version of that latest code that they can then play around with and edit however they like.